packages/graphql/src/graphql.ts

```diff
@@ -79,14 +60,14 @@ type InternalData = {
 }
 
 function typeToGraphQLOutputTypeInternal(type: types.Type, internalData: InternalData): GraphQLOutputType {
-  const { inspectedTypes, knownTypes, defaultName } = internalData
+  const { inspectedTypes, knownOutputTypes, defaultName } = internalData
   // If the type has already been explored, then return the output type that has
   // already been generated
   if (inspectedTypes.has(type)) {
```

Comment:
Here I tried adding an additional condition `&& knownOutputTypes.has(type)` but it resulted in a stack overflow and I have no idea why